ui/lib/interfaces.ts

```diff
@@ -7,6 +7,8 @@ export interface BaseRecord {
 export interface Habit extends BaseRecord {
   title: string
   description: string
+  frequency: string
```

Comment:
Create a type for this in the frontend (not an enum):
`export type frequency = 'DAILY | 'WEEKLY' | 'MONTHLY' | 'YEARLY'`